src/tech/v3/dataset/impl/column_base.clj

```diff
@@ -53,6 +57,11 @@
              (dotimes [iter n-elems]
                (.add list-data nil))
              list-data)
+     :mmap-string (mmap-str/->MmapStringList "/tmp/test.mmap"
```

Comment:
Obviously this need to be change here. A tempory file would work.

But I have the uses case to  read text from various files, and it would be nice to be able to use a single mmap file 
 Or find a way to use a single MmapStringllist across various data frames ...
  But it might be better to have a special handling of MmapStringLists at teh moment we `concat` datasets.
  Or a specific method to "compact" various MmapStringLists 
 
 Or give the user a way to manage and configure" the concrete MmmapStringList to use at any `->dataset` operation

Reply:
I agree and I was also stuck on that exact point.  I think the answer to that is to allow a per-column option during parsing to specify the file to use as a backing store.